cpu/esp32/startup.c

```diff
@@ -336,6 +327,13 @@ static NORETURN void IRAM system_init (void)
     /* trigger static peripheral initialization */
     periph_init();
 
+    /* print current system time after RTC peripherals are initialized */
+    struct tm _sys_time;
+    rtc_get_time(&_sys_time);
+    ets_printf("System time: %04d-%02d-%02d %02d:%02d:%02d\n",
+               _sys_time.tm_year + 1900, _sys_time.tm_mon + 1, _sys_time.tm_mday,
+               _sys_time.tm_hour, _sys_time.tm_min, _sys_time.tm_sec);
+
```

Comment:
Do we really need that noise?

Reply:
I guess it's a question of taste. I like that a system informs me about the configuration and the settings instead of starting silent and the user doesn't know whether it hangs or it runs.

Reply:
There is value in the Unix way of only printing something when necessary. If there is always lots of output you'll for the most part ignore, so it will hide important things.

I didn't see `SLAAC not activated; will not auto-configure IPv6 address for interface 9.` initially because there is just so much printed during startup.

Reply:
Hm, for me these information are useful. For example, I wish that the board configuration would be printed for all boards instead of browsing through the board definition files each time to know which pins are being used for what.